questions.md

```diff
@@ -0,0 +1,3 @@
+Could you tell difference between pull and fork?
```

Comment:
A `pull` command is both a fetch and a merge of the project (for a project you are planning to actively contribute to).
A fork is a copy of the parent project, that you may use to create a pull request.